src/main/kotlin/app/revanced/patches/remini/misc/premium/fingerprints/PremiumUnlockFingerprint.kt

```diff
@@ -0,0 +1,16 @@
+package app.revanced.patches.remini.misc.premium.fingerprints
+
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.Opcode
+
+object PremiumUnlockFingerprint : MethodFingerprint(
+    opcodes = listOf(
+        Opcode.INVOKE_VIRTUAL,
+        Opcode.MOVE_RESULT_OBJECT,
+        Opcode.IGET_BOOLEAN,
+        Opcode.IF_EQZ,
+        Opcode.GOTO,
+        Opcode.CONST_4
+    ),
+    customFingerprint = { it.definingClass == "Lna/k;" && it.name == "o" }
```

Comment:
This should be improved so future versions work with the patch.

Reply:
I don't think the patch will work on newer/older versions at any circunstances since the package names got renamed twice, at least, across versions

Reply:
But what can I do for better compatibility?

Reply:
If the symbols change, you can create a fingerprint with an opcode pattern or strings and so on, theres multiple properties you can use to find a method without names, regardless in which package they are.

Reply:
I'll try to make it